python_modules/libraries/dagster-tableau/dagster_tableau/resources.py

```diff
@@ -303,36 +381,56 @@ def fetch_tableau_workspace_data(
 
     def build_assets(
         self,
+        start_workbook_refresh_requests: Sequence[StartWorkbookRefreshRequest],
         dagster_tableau_translator: Type[DagsterTableauTranslator],
     ) -> Sequence[CacheableAssetsDefinition]:
         """Returns a set of CacheableAssetsDefinition which will load Tableau content from
         the workspace and translates it into AssetSpecs, using the provided translator.
 
         Args:
+            start_workbook_refresh_requests (Sequence[StartWorkbookRefreshRequest]): A list of
```

Comment:
I think it's a bit unclear from the docstring what providing this argument does - this creates an SDA for each workbook associated with a request, right?

In general I think this name is a bit confusing. The user is submitting a "template" for a request and not a request itself (since the requests are made at runtime). Moreover, each request passed here instructs Dagster to build an SDA to allow the workbook to be refreshed. This is not very clearly represented in the name/docstring.

I had similar feedback for the Looker equivalent (`RequestStartPdtBuild`), which I guess this is modeled after. In that case the justification was that calling it `RequestStartPdtBuild` matched the name in documentation so it was easier for users to understand the different configuration options and to look up docs. In this case, since the only argument is the workbook ID, I don't think we get the same utility out of retaining the name from Tableau's API.

I'd suggest calling it something like `enable_refreshing_workbooks` and take a sequence of workbook IDs, unless we expect to add other parameters down the line. Even if we retain the dataclass name, I think the argument or docstring could be updated to be more clear, but curious for your thoughts.

Reply:
Wrote this up before seeing Ben's parallel comment:
---
I know it's built to be parallel to a similar concept in the Looker integration, but the name `StartWorkbookRefreshRequest` strikes me as a little odd – it's a verb if I'm reading it correctly, but classes are normally nouns. Two thoughts:
- Is there a shorter noun that would make more sense?
- Do we need a class for this at all? E.g. maybe just something like `refreshable_workbook_ids`?

Reply:
I also think we should consider changing the name in Looker.

Reply:
That makes a lot of sense. This is modeled after the Looker integration, but with this feedback I agree that we should make the changes in both integrations.

Reply:
Changes made in [3c010c6](https://github.com/dagster-io/dagster/pull/24862/commits/3c010c64b1b5b07b3724aacbd4b833cc893152f6)